pandas/formats/format.py

```diff
@@ -197,7 +197,8 @@ def _get_footer(self):
                                        escape_chars=('\t', '\r', '\n'))
             footer += ("Name: %s" % series_name) if name is not None else ""
 
-        if self.length:
+        if (self.length is True or
```

Comment:
should have a check on the valid values of this (even though the option actually vets this)